src/main/kotlin/com/jaoafa/vcspeaker/events/ReadyEvent.kt

```diff
@@ -13,6 +14,10 @@ class ReadyEvent : Extension() {
         event<ReadyEvent> {
             action {
                 logger.info { "Ready! Logged in as ${event.self.tag}." }
+
+                event.kord.editPresence {
+                    watching("users talk | VCSpeaker ${VCSpeaker.version}")
```

Comment:
`users talk` いるかな…？ステータスメッセージが長すぎると、見え方として切れてしまいそうで
あと、プロジェクト名が `VCSpeaker.kt` なので、`VCSpeaker.kt {version}` で良いようなきがしています。

Reply:
Watching VCSpeaker.kt {version} って変じゃありません...?
そもバージョンのことを気にするのは開発者くらいなもので、一応確認できればいいレベルの情報なので、ホバーして見るくらいでいいと思うのですが

Reply:
あーそういう見え方になるのか…、じゃあいいかもです。
ただ、ユーザー側に見える情報になるので、さきほど整理したように以下でどうでしょう。
```suggestion
                    watching("users talk | VCSpeaker.kt ${VCSpeaker.version}")
```

Reply:
適用しました ✅